www/src/layouts/landingPage.astro

```diff
@@ -21,17 +21,23 @@ const currentPage = Astro.url.pathname;
     <title>{SITE.title}</title>
   </head>
   <body class="bg-gradient-to-b from-gray-900 via-[#300171] to-slate-900">
-    <Navbar />
+    <img
+      src="/images/background-pattern.png"
```

Comment:
I like how this looks, but shipping a 225kb png for something so subtle seems excessive. Is it possible reduce the file size? If compression doesn't do it, maybe just smaller dimensions?

Reply:
Hmm I'm gonna try exporting it as SVG

Reply:
Could also make it a webp

Reply:
nice, its over 90% smaller as an svg :)